content/en/agent/guide/agent-configuration-files.md

```diff
@@ -73,14 +73,16 @@ Prior releases of Datadog Agent stored configuration files in `/dd-agent/conf.d/
 
 ### Checks configuration files for Agent 6
 
-An example for each Agent check configuration files can be found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` in order to enable the associated check. Note that the Agent loads any valid YAML file contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This way, complex configurations can be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
+An example for each Agent check configuration file can be found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` in order to enable the associated check. Note that the Agent loads  valid YAML file contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This way, complex configurations can be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
 
 ```text
 /etc/datadog-agent/conf.d/http_check.d/
 ├── backend.yaml
 └── frontend.yaml
 ```
 
+A special case are YAML files with the suffix `.default`.  These files will be loaded by the Agent automatically and help define the core set of checks that are always enabled.  To turn off one of these checks, simply change the suffix to `.disabled` or remove the file altogether.
+
```

Comment:
The main point is that they're only loaded if we can't find a configuration for the check. So if the user created a disk.yaml the disk.yaml.default won't be loaded. User should not have to remove or renamed them (the goal was for them to be mostly ignored by users).

Reply:
Makes sense, added a line suggesting the user copy the `.default` file to make any changes.